scipy/stats/_multivariate.py

```diff
@@ -1448,6 +1449,626 @@ def entropy(self):
+    Among-column covariance matrix of the distribution (default: ``1```)
+df : scalar, optional
+    Degrees of freedom (default: ```1```)
+"""
+
+_matt_doc_callparams_note = """\
+If `mean` is set to `None` then a matrix of zeros is used for the mean.
+The dimensions of this matrix are inferred from the shape of `rowcov` and
+`colcov`, if these are provided, or set to ``1`` if ambiguous.
+
+`rowcov` and `colcov` can be two-dimensional array_likes specifying the
+covariance matrices directly. Alternatively, a one-dimensional array will
+be be interpreted as the entries of a diagonal matrix, and a scalar or
+zero-dimensional array will be interpreted as this value times the
+identity matrix.
+"""
+
+_matt_doc_frozen_callparams = ""
+
+_matt_doc_frozen_callparams_note = """\
+See class definition for a detailed description of parameters."""
+
+matrix_t_docdict_params = {
+    "_matt_doc_default_callparams": _matt_doc_default_callparams,
+    "_matt_doc_callparams_note": _matt_doc_callparams_note,
+    "_doc_random_state": _doc_random_state,
+}
+
+matrix_t_docdict_noparams = {
+    "_matt_doc_default_callparams": _matt_doc_frozen_callparams,
+    "_matt_doc_callparams_note": _matt_doc_frozen_callparams_note,
+    "_doc_random_state": _doc_random_state,
+}
+
+
+class matrix_t_gen(multi_rv_generic):
+    """
+    A matrix t-random variable.
+
+    The `mean` keyword specifies the mean.
+    The `rowcov` keyword specifies the among-row covariance matrix.
+    The `colcov` keyword specifies the among-column covariance matrix.
+
+    Methods
+    -------
+    pdf(x, mean=None, rowcov=None, colcov=None)
+        Probability density function.
+    logpdf(x, mean=None, rowcov=None, colcov=None)
+        Log of the probability density function.
+    rvs(mean=None, rowcov=1, colcov=1, df=1, size=1, random_state=None)
+        Draw random samples.
+    entropy(df, rowcov=None, colcov=None)
+        Differential entropy of the probability density function.
+
+    Parameters
+    ----------
+    %(_matt_doc_default_callparams)s
+    %(_doc_random_state)s
+
+    Notes
+    -----
+    %(_matt_doc_callparams_note)s
+
+    The covariance matrices specified by `rowcov` and `colcov` must be
+    (symmetric) positive definite. If the samples in `X` have shape `(m,n)`
+    then `rowcov` must have shape `(m,m)` and `colcov` must have shape `(n,n)`.
+    Covariance matrices must be full rank.
+
+    The probability density function for `matrix_t` is
+
+    .. math::
+        \mathcal{T}_{m,n} = \\frac{\Gamma_n \left( \\frac{\mathrm{df} + m + n - 1}{2} \\right)}{
+        \Gamma_n \left( \\frac{\mathrm{df} + n - 1}{2} \\right)\pi^{mn / 2}\left|\Sigma\\right|^{n/2}
+        \left|\Omega\\right|^{m/2}}\left|I_n + ( X - \mathrm{M} )^T \Sigma^{-1} ( X - \mathrm{M} ) \Omega^{-1}\\right|^{
+        -\\frac{\mathrm{df} + m + n - 1}{2}}
+
+    or, alternatively,
+
+    .. math::
+        \mathcal{T}_{m,n} = \\frac{\Gamma_m \left( \\frac{\mathrm{df} + m + n - 1}{2} \\right)}{
```

Comment:
at a guess, this may fix most of the CI failures.

```suggestion
    .. math::

        \mathcal{T}_{m,n} = \\frac{\Gamma_n \left( \\frac{\mathrm{df} + m + n - 1}{2} \\right)}{
        \Gamma_n \left( \\frac{\mathrm{df} + n - 1}{2} \\right)\pi^{mn / 2}\left|\Sigma\\right|^{n/2}
        \left|\Omega\\right|^{m/2}}\left|I_n + ( X - \mathrm{M} )^T \Sigma^{-1} ( X - \mathrm{M} ) \Omega^{-1}\\right|^{
        -\\frac{\mathrm{df} + m + n - 1}{2}}

    or, alternatively,

    .. math::

        \mathcal{T}_{m,n} = \\frac{\Gamma_m \left( \\frac{\mathrm{df} + m + n - 1}{2} \\right)}{
```

Reply:
Thanks for the tip. Docstrings are updated and and docs build/render correctly locally.